flang/test/Lower/array-expression.f90

```diff
@@ -426,4 +426,18 @@ subroutine test18
   row_i => array(i, :)
 end subroutine test18
 
+! CHECK-LABEL: func @_QPtest_column_and_row_order(
+subroutine test_column_and_row_order(x)
```

Comment:
You do not need a CHECK-SAME on x (e.g., %arg0) ?

Reply:
No because there is no need to capture the argument in this test. The test is really only focused on the induction variables/loop order, not really what the loops are doing and on what.
There is always a trade of in lit tests between capturing enough patterns to demonstrate a property of the lowered IR and avoiding capturing too much to avoid corelating the test with other IR properties that are not central to the test and would break the test if changed. Here, argument lowering is not something needed for the test, so I rather not add any related checks.